### PR TITLE
BTA-6752 Update API Documentation with USD::Bank

### PR DESCRIPTION
**Pull request by cesarecamurani**
[BTA-6752](https://azafinance.atlassian.net/browse/BTA-6752): [USD::Bank - Glo Currency ] - Update API documentation, API Spec and SDKs

Related to: https://github.com/bitpesa/bitpesa-api-specification/pull/263

Changes
---------
- Added `USD::Bank` to the payout details page.

Screenshot 2021-06-09 at 15 32 46
Screenshot 2021-06-11 at 08 13 49

### CODE REVIEW

File: _docs/payout-details.md [hunk 1]
```diff
@@ -87,6 +87,40 @@ For Nigerian mobile payments please use:
...
+
+{% include language-tabbar.html prefix="usd-bank-details" raw=data-raw %}
+
+The valid `bank_code` values are:
+
+{% capture data-raw %}
+```
+FCMB Bank: 214
+United Bank for Africa: 033
+Zenith International: 057
```
Comment (mchochowski) [inline _docs/payout-details.md:114]: I'm not sure if we should mention Zenith, see last message in #glo-currency-integration
channel.

Comment (cesarecamurani) [reply _docs/payout-details.md:114]: Yes, I've seen it... I was waiting for feedbacks from Derrick but I think we can assume it won't be available for now, I'll remove it...

**Review by mchochowski — DISMISSED** [review]

File: _docs/payout-details.md [hunk 2]
```diff
@@ -87,6 +87,41 @@ For Nigerian mobile payments please use:
 **Warning!** The recipient has to be registered and KYC'd with <https://www.mypaga.com>{: .alert-link} before they can receive funds. If they are not registered when the payment occurs then Paga can hold the funds until the user registers and approves themselves.
 </div>
 
+## USD::Bank
+
+For USD to USD bank payments in Nigeria please use:
```
Comment (CallumD) [inline _docs/payout-details.md:92]:
```suggestion
For USD bank payments in Nigeria please use:
```

**Review by CallumD — CHANGES_REQUESTED** [review]

**Review by CallumD — APPROVED** [review]
@sztupy @jara @aniki21 FYI

File: _docs/payout-details.md (hunk 1 above)
Comment (aniki21) [reply _docs/payout-details.md:115]: @cesarecamurani Is Zenith being removed, in the end?

Comment (cesarecamurani) [reply _docs/payout-details.md:115]: It's back apparently... https://azafinance.slack.com/archives/C021UK3RZDJ/p1623320822004100

**Review by aniki21 — APPROVED** [review]

**Pull request merged**